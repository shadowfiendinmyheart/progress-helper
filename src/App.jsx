import React from "react";
import {BrowserRouter as Router} from "react-router-dom";
import Header from "./components/Header";
import AuthRouter from "./routers/AuthRouter";

const App = () => {
  return (
    <Router>
      <Header />
      <AuthRouter />
    </Router>
  );
};

export default App;
