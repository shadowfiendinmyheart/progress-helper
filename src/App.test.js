import {render, screen} from "@testing-library/react";
import "@testing-library/jest-dom";
import App from "./App";

describe("test app component", () => {
  test("render headers", () => {
    render(<App />);
    // const h1Element = screen.getByText(/Hello world!!!/i);
    // expect(h1Element).toBeInTheDocument();
    // expect(h1Element).toMatchSnapshot();

    // const h2Element = screen.getByText(/Just perfect!/i);
    // expect(h2Element).toBeInTheDocument();
    // expect(h2Element).toMatchSnapshot();

    // const h3Element = screen.getByText(/Good/i);
    // expect(h3Element).toBeInTheDocument();
    // expect(h3Element).toMatchSnapshot();
  });
});
