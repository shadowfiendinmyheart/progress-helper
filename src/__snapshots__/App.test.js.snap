// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app component tests 1`] = `
<h1
  class="app"
>
  Hello world!!!
</h1>
`;

exports[`app component tests 2`] = `
<h2>
  Just perfect!!
</h2>
`;

exports[`app component tests 3`] = `
<h3>
  Good!
</h3>
`;
